.github/workflows/reusable-revised-its.yml

```diff
@@ -117,8 +117,11 @@ jobs:
           echo $DRUID_IT_IMAGE_NAME
           docker save "$DRUID_IT_IMAGE_NAME" | gzip > druid-container-jdk${{ inputs.build_jdk }}.tar.gz
 
-      - name: Load docker image
+      - name: Clean docker environment and Load docker image
         run: |
+          echo "Stopping and pruning any remnant docker containers from previous run"
+          docker ps -aq | xargs -r docker rm -f
+          docker system prune -af --volumes
```

Comment:
could this be done in the script itself? So that anyone running an IT test locally would not run into this problem as well.

Reply:
I had a PR open for this here: https://github.com/apache/druid/pull/13940
If you read Paul's comment here https://github.com/apache/druid/pull/13940/files#r1139411707, he feels the same way.  I agree, it should probably be rolled in.  It's also less heavy handed than this, since this removes ALL containers on the runner, which includes some pre-cached ones that save a little time.  Honestly though, they're not that useful for us and the cleanup is very brief, so I didn't put much weight on being less heavy handed for the older ITs, given they're not long for the world.

Reply:
Oh, I'm sorry, I didn't know you had an existing PR. I'll remove this.

Reply:
No problem, it's still an "open issue", so let's see if we can get a final call one way or the other.  :)